cmd/helm/install.go

```diff
@@ -181,10 +183,19 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
-	// Check chart dependencies to make sure all are present in /charts
-	chartRequested, err := loader.Load(cp)
-	if err != nil {
-		return nil, err
+	// Check chart dependencies to make sure all are present in the InMemoryCache or in /charts
+	chartRequested := &chart.Chart{}
+	if client.UseInMemoryCache {
+		if _, err := os.Stat(cp); err == nil {
+			chartRequested, err = loader.Load(cp)
+		} else {
+			chartRequested = downloader.InMemoryCache[schart].Chart
```

Comment:
This does not compile.  There is no `Chart` field in you latest version of #7601